recipes/lovely-jax/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "lovely-jax" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lovely-jax-{{ version }}.tar.gz
+  sha256: 605ab1c0b15bb97615d64444bda2f8f076b03157831f84c50499a916b6c9469b
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - fastcore
+    - jax
+    # missing dependency
+    - matplotlib
```

The base variant (no Qt) is typically enough.
```suggestion
    - matplotlib-base
```